### PR TITLE
Modifications to New Deep jobs

### PR DESCRIPTION
**Pull request by Anarchist2**
This PR fixes some of the errors present in the jobs committed in 449debe59be4476707ea12a572ec35f7ea6f6ec2, most notably the ability for jobs to have the same destination as a source. It also fixes some of the text substitutions to the correct ones and changes some of the text, such as making it more explicit that the consultants leave behind stuff on your ship and pointing out how time-sensitive the "urgent" missions are. Also included are changes to the payment amounts of each job, integration of rep mechanics to the older mystery cube jobs, and removal of reputation penalties for the jobs, due to being problematic if the player fails/aborts them while at low reputation.

### CODE REVIEW

**Comment by Zitchas** [comment]
Ah, doesn't `distance 1` force the destination to be exactly 1 step away?

I think we might actually want something more like `distance 1 8` That should allow it to cover the entirety of the Deep while excluding the place the mission is coming from.

**Review by Amazinite — CHANGES_REQUESTED** [review]

File: data/human/deep jobs.txt
Comment (Terin) [reply data/human/deep jobs.txt:411]: `distance 2 7` feels like an appropriate amount, ferrying things over a single system is a little tiny for starters, and then 7 is the max, yeah.

**Comment by Terin** [comment]
This isn't quite in the scope of fixing bugs for these missions, but the payments need to be adjusted, at least for the starter missions.

I was able to pay off my loan in two and a half minutes, with nothing but these 1 ton, no deadline, no danger missions that paid out 60-110k credits. `payment 15000 2500` felt a lot more appropriate for someone coming into Deep space with no previous experience, and having the payment go up when you get in their good graces (and do Mystery Cube jobs, which require you to get in danger) is a decent reward.

**Comment by Anarchist2** [comment]
I've lowered most of the payments for the jobs by significant amounts. Some points of comparison:
* The initial and basic Secure Transport currently has `payment 50000 10000`. This equates to 70k credits at a distance of 1, and 130k at a distance of 7. To compare, the average Large Bulk Delivery job, consisting of 98 tons of cargo, pays 50.1k at the minimum distance of 2, and 138.3k at 8 jumps.
  * This PR lowers the payment to `payment 3000 900`, paying 5.2k at 2 jumps and 10.2k at 7. Compare with Paradise Job: Care Package, paying 5.9k at 5 jumps and 1 ton + deadline, Paradise Job: Birthday Supplies, paying 8.3k at 10 jumps and 2 tons + deadline, Paradise Job: Art Delivery, paying 11.7k at 5 jumps and 5 tons + deadline, Record academic conference, paying 16k credits at 10 jumps and 1 ton, and Mystery Cube Delivery Job (and Retrieval), paying 26k at 20 jumps and 1 ton + bounty hunters.
* The DS Secure Transport jobs have `payment 80000 15000`. This is 110k at 1 jump, and 200k at 7 jumps. In comparison, the largest Frontier Delivery job, with 98 tons of cargo + 3 pirate fleets attacking you, pays 56.9k at 2 jumps and 198k at 10 jumps.
  * The payment has been changed to `payment 5000 1500`, paying 9.5k at 2 jumps and 17k at 7. Compare to Paradise Job: Theater Props, paying 24.2k at 7 jumps and 12 tons + deadline, and Paradise Job: Fine Food, paying 36k at 10 jumps and 17 tons + deadline.
* Transport Consultants has `payment 200000 35000`, equivalent to 5.8 million credits at 1 jump and 22.7 mil at 7 jumps, with the maximum passenger amount of 8.
  * This PR changes the payment to `payment 10000 600`, paying 100k credits at 2 jumps and 394k at 7, with a compliment of 8 passengers. Compare with Strike breakers, which pays 250k credits at 7 jumps and 20 passengers.
* Urgent Deep Sec Package pays with `payment 200000 30000`, which is 260k at 1 jump and 440k at 7.
  * Payment is now `payment 0 300000`, paying 90k at 2 jumps and 240k at 7. It's also locked behind the "deep security reveal" condition now.
* Urgent Deep Sec Consultants has `payment 300000 3000`, paying out 660k credits at 1 jump and 1.7 million at 7 jumps, with the maximum passenger amount of 6.
  * Payment has been dropped to `payment 150000 1500`, paying 420k at 2 jumps and 870k credits at 7 jumps, with 6 passengers. Compare with the Field Trip job, which pays 730k at 7 jumps and 60 passengers. Additionally, encounter rate for both missions have dropped to 1%, and are both blocked behind "deep security consultants".

**Comment by Anarchist2** [comment]
Thinking about it more, reputation penalties for failing these jobs could probably still be kept if multipliers are used instead of just subtraction. For instance, failing an initial secure package job could multiply your rep by 0.9, resulting in a -1 rep at 10 rep, -2 at 20, etc. while also making it impossible to ever be hostile with Deep Security purely through the jobs. Still not keen on the idea of impacting rep on failure, as opposed to something like subtracting from the "secure deep package" condition.

**Review by Amazinite — CHANGES_REQUESTED** [review]
Not sure why the multipliers were so high to begin with, but these multipliers still seem too high.

The default multiplier on most jobs in human space is 150.
Regional human jobs have multipliers of 160 to 180 (with some at 50 to 140 for flavor).
FW war jobs have multipliers of 180.
Higher end jobs (e.g. wealthy passenger transportation) have multipliers of 200 to 250.
There's one Wanderer job and two Hai jobs with a multiplier of 300.
Above a multiplier of 300, the original mystery cube delivery jobs and the "Record academic conference" job, both of which involve only transporting 1 ton of cargo, have a multiplier of 1000, and the Unfettered Aid and Tribute jobs are the major outliers with multipliers of 1600-2000 and 1600-2400 respectively. 
(Illegal pirate jobs have multipliers from 300 to 1000, but the increased payment there is for the danger of running them.)

* The early secure transport jobs seem fine at 900 since they're similar to the mystery cube jobs (transporting 1 ton) but lack bounty hunters.
* Not so sure on the later secure transport jobs having a multiplier of 1500 with no risk; that seems like quite the jump from 900. Perhaps lower the earlier ones from 900 to 750 and the later ones from 1500 to 1000.
* I'd drop the passenger jobs from 600 to 300 at the most, as that's still a higher modifier than anything else in human space that isn't a pirate job or another Deep job.
* I'll have to keep my eye on that urgent package one.
* Drop the urgent consultant jobs to 300 as well; the guaranteed 150k (which may be too high over the previous guaranteed 10k) is enough of an increased payment that we don't need these jobs to have a better multiplier (and other rush jobs typically use a higher base pay than a higher multiplier anyway).

File: data/human/deep jobs.txt
Comment (Amazinite) [inline data/human/deep jobs.txt:878]: I think these last three jobs still have way too high base payments. 170k or 450k guaranteed for 6-7 jumps of not refueling doesn't match the difficulty, as these are really only difficult to first time you take them, then after that you should be set to complete any of them without issue. For comparison, 150k is what you get for tracking down and killing a medium warship bounty, and 450k is what you get for taking on a Marauder V bounty, which is a Marauder medium warship and two lights.

```suggestion
		payment 30000 1000
```

File: data/human/deep jobs.txt
Comment (Amazinite) [inline data/human/deep jobs.txt:930]:
```suggestion
		payment 60000 300
```

**Review by Amazinite — CHANGES_REQUESTED** [review]

**Comment by MasterOfGrey** [comment]
> This isn't quite in the scope of fixing bugs for these missions, but the payments need to be adjusted, at least for the starter missions.
> 
> I was able to pay off my loan in two and a half minutes, with nothing but these 1 ton, no deadline, no danger missions that paid out 60-110k credits. `payment 15000 2500` felt a lot more appropriate for someone coming into Deep space with no previous experience, and having the payment go up when you get in their good graces (and do Mystery Cube jobs, which require you to get in danger) is a decent reward.

The payments were set higher because these jobs encourage you to stay around long enough to stumble upon the various bactrian questline missions, so they need to be enticing enough for you to stay instead of stocking up on jobs to other corners of the galaxy and leaving again. The **should** be a little more profitable than expected, because the trade-off is you're staying in one little corner of the galaxy and not hitting other content.

_Having said that_ I didn't spend a lot of time testing them in-game for suitability, I just looked at like two of them and they seemed reasonable, probably luck at the time.

I'm in support of them being brought down to just a little above average, and then the rewards only being higher for the ones that require you to get into the Deep's good graces first.

...
I do not particularly support changing the reputation penalties though. These jobs are easy, if you're messing them up that should be viewed as malicious disregard for important Deep affairs.

File: data/human/deep jobs.txt
Comment (MasterOfGrey) [inline data/human/deep jobs.txt:409]: Why have these been removed from the Source? I understand the validity of also excluding them from the destination, but I would expect they need to remain in the source as well.

Comment (Zitchas) [reply data/human/deep jobs.txt:409]: Is there a reason why they can't be offered from moons or stations?

Comment (MasterOfGrey) [reply data/human/deep jobs.txt:409]: Mostly just none of the texts make sense for them.

**Comment by Anarchist2** [comment]
> These jobs are easy, if you're messing them up that should be viewed as malicious disregard for important Deep affairs.

I can certainly see that line of thinking working for failing the missions, but it feels far too harsh of a penalty for aborting. Consider the following:
* A new player ends up in the Deep.
* Looking through the job board, they accept a "Secure Transport" job.
* However, at the bottom of the list, they find a lucrative family transport job, heading all the way to the dirt belt. They accept that instead.
* The Secure job is aborted in order to fit more cargo for jobs that are on the way to the family transport destination, or just for simple commodities hauling.
* They take off and are shot at out of seemingly nowhere.
* For extra fun and profit, feel free to accept more secure transport jobs, even as Deep Security tries to kill you.

File: data/human/deep jobs.txt
Comment (Amazinite) [inline data/human/deep jobs.txt:413]: Isn't this line redundant with the one above it? If your reputation is above 0 then it's never negative, so the second line doesn't matter. If your reputation is negative, then you'll be blocked by the first line.

File: data/human/deep jobs.txt
Comment (Amazinite) [inline data/human/deep jobs.txt:431]: Make this change to any mission with an `on fail`. I don't support aborting missions causing a negative reputation hit, and I have no idea why not only did the PR that added these jobs have that but also why it punished aborting a mission more than failing it, as failing a mission can mean outright losing the cargo or passengers do to a destroyed ship; I imagine the Deep would be more concerned with loss of cargo or personnel than you just leaving them at a spaceport somewhere.
```suggestion
	on abort
	on fail
```

**Review by Amazinite — CHANGES_REQUESTED** [review]

File: data/human/deep jobs.txt
Comment (Anarchist2) [reply data/human/deep jobs.txt:413]: The first line is a sanity check to make sure that Deep Sec aren't offering jobs to someone they hate. The second one is to make players who've failed a secure transport job need to raise their rep before they can be trusted another one again (and that line would allow negative rep players to still be offered secure transports if "secure deep package" is positive).

Comment (Amazinite) [reply data/human/deep jobs.txt:413]: Sounds like you're missing an `or`.

Comment (Anarchist2) [reply data/human/deep jobs.txt:413]: My thought process behind having both conditions:
* When "secure deep package" is 0: `- "secure deep package" * 3` is also 0, so the player is offered these missions when above negative rep.
* When "secure deep package" is positive: `- "secure deep package" * 3` is negative, but the first line stops players with negative rep from being offered the job anyway.
* When "secure deep package" is negative (through failing jobs): `- "secure deep package" * 3` is positive, meaning that players who've failed a secure transport job need to increase their rep with Deep Security before they can be offered the missions again.

Comment (Amazinite) [reply data/human/deep jobs.txt:413]: That checks out, then.

**Comment by MasterOfGrey** [comment]
> * The Secure job is aborted in order to fit more cargo for jobs that are on the way to the family transport destination, or just for simple commodities hauling.

Ok I see this point for aborting them, shame we can't test the "aborted in space"?
I still think they absolutely need to keep the penalty for failing though.

**Comment by Amazinite** [comment]
Aborting missions in space doesn't immediately kick the cargo and passengers out the door. You keep them until landing.

**Pull request merged**
